src/main.rs

```diff
@@ -516,6 +518,7 @@ mod unit_tests {
                 rng: RngConfig {
                     src: PathBuf::from("/dev/urandom"),
                     iommu: false,
+                    id: None,
```

Comment:
For things like rng where is is static i.e. `_rng0` then it doesn't need to be  member.